style.css

```diff
@@ -538,3 +531,251 @@ body {
   border-radius: 8px;
   border: 1px solid #cfd8dc;
 }
+
+#submit:hover {
+  background: #6070ff;
+  color: #fff;
+}
+
+#submit:active {
+  background: #2230d2;
+  color: #fff;
+}
+
+#submit:disabled {
+  background: #c1c7d0;
+  color: #fff;
+}
+
+.second {
+  display: none;
+}
+
+.two {
+  display: none;
+}
+
+@media screen  and (min-width: 768px) {
+  body {
+    background: #e5e5e5;
+  }
+
+  .home-page-section {
+    background: #fff;
+    height: 916px;
+    width: 1440px;
```

Comment:
- [ ] Please remove fixed widths in your code since they don't work on different screen sizes and not a good practice. Right now, your page width is set to 1440px. And it overflows the screens smaller than 1440px and gives the user horizantal scrolling.  We should use flex and grid to layout our elements and margins and paddings for setting spacings between them. Here's a screenshot for given template: 

![image](https://user-images.githubusercontent.com/91486791/193141779-ad64c053-cd7a-494a-96b4-5927b5cc57d4.png)
And this is how your work look:

![image](https://user-images.githubusercontent.com/91486791/193142116-d88cf700-0377-44af-8e94-5e25e9d9fac8.png)